xml/windows-ses.xml

```diff
@@ -120,7 +120,7 @@
   <para>
    &ceph; for &mswin; can be easily installed through the
    <filename>SES4Win.msi</filename> setup wizard. You can download this from
-   <link xlink:href="https://beta.suse.com/private/SLE15/SP2/download/SES7/SES4Win/">SES4Win</link>.
+   <link xlink:href="https://beta.suse.com/private/SLE15/SP3/download/SES7/SES4Win/">SES4Win</link>.
```

Comment:
```suggestion
   <link xlink:href="https://beta.suse.com/private/SLE15/SP3/download/SES7.1/SES4Win/">SES4Win</link>.
```

@ml8mr I think this will be rebuilt too?